.github/workflows/nodejs.yml

```
@@ -1,6 +1,17 @@
 name: Node CI
 
-on: [push]
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - client/
```

```suggestion
    - client/*
```
If I understand the docs correctly. https://help.github.com/en/articles/workflow-syntax-for-github-actions